extensions/2.0/Vendor/EXT_mesh_features/schema/primitive.EXT_mesh_features.schema.json

```diff
@@ -1,16 +1,26 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "primitive.EXT_mesh_features.schema.json",
   "title": "EXT_mesh_features glTF Primitive extension",
   "type": "object",
-  "description": "`EXT_mesh_features extension` for a primitive in a glTF model, to associate it with the root `EXT_mesh_features` object.",
+  "description": "`EXT_mesh_features` extension for a primitive in a glTF model, to associate it with the root `EXT_mesh_features` object.",
+  "allOf": [
+    {
+      "$ref": "glTFProperty.schema.json"
+    }
+  ],
   "properties": {
     "featureIds": {
       "type": "array",
-      "description": "",
+      "description": "An array of feature IDs. (TODO: one to one mapping with `propertyTables`)",
```

Comment:
This description was empty. @donmccurdy do you have any suggested wording here?

Reply:
```suggestion
      "description": "An array of feature IDs. A property table at index `i` corresponds to the `featureIds` entry at the same index. Additional feature ID entries may be present, so the length of the `featureIds` array must be greater than or equal to the length of the `propertyTables` array.",
```